rockcraft/models/project.py

```diff
@@ -306,7 +280,7 @@ def _check_unsupported_options(cls, values: Mapping[str, Any]) -> Mapping[str, A
 
         return values
 
-    @pydantic.validator("license", always=True)
+    @pydantic.field_validator("license")
```

Comment:
Thoughts on using https://github.com/canonical/craft-application/pull/402 for this?

Reply:
I think it's a good idea, but as follow-up work